.gitignore

```diff
@@ -30,3 +30,7 @@ Session.vim
 
 # User cluster configs
 .kubeconfig
+
+#files generated by cve feed prow job
+sig-security-tooling/cve-feed/hack/cve-feed-hash
+sig-security-tooling/cve-feed/hack/official-cve-feed.json
```

Comment:
Can we create a separate `.gitignore` file for `sig-security-tooling/hack` sub-directory because:
- It will allow the tooling OWNERS to manage the changes specific to tooling files to ignore
- Approvals to the PR will be easier
- Other sub-projects do not have to worry about these files

Reply:
Updated